### ISSUE TITLE
Reproduce the results on CoLA and STS-B

### ISSUE THREAD

**Issue by cooelf** [google-research/bert#505]
I try to reproduce the CoLA and STS-B results reported in the BERT paper but the numbers are far from the reported one. The followings are my best results.

|       | learning rate | num_epochs | dev         | test      | report    |
| ----- | ------------- | ---------- | ----------- | --------- | --------- |
| CoLA  | 2e-5          | 5          | 64.79       | 56.9      | 60.5      |
| STS-B | 2e-5          | 5          | 90.42/90.04 | 83.0/81.2 | 87.6/86.5 |

The code for STS-B is from #74 https://github.com/Colanim/BERT_STS-B/blob/master/run_scorer.py

As the paper said,
for BERT_LARGE we found that fine-tuning was sometimes unstable on small data sets (i.e., some runs would produce degenerate results), so we ran several random restarts and selected the model that performed best on the Dev set.

I also tried several restarts with different learning rates and random seeds but it seems no improvement. I'm quite confused for the reproduction. Any suggestions would be greatly appreciated.
